charts/README.md

```diff
@@ -38,13 +39,14 @@ deployments. All the options are documented in the configuration section.
 Please refer to the release notes of each version of the helm charts.
 These can be found [here](https://github.com/kubewarden/helm-charts/releases).
 
-## Uninstalling the chart
+## Uninstalling the charts
 
-To uninstall/delete the `kubewarden-controller` release use the following
+To uninstall/delete kubewarden-controller and kubewarden-crds use the following
 command:
 
 ```console
 $ helm uninstall -n kubewarden kubewarden-controller
+$ helm uninstall -n kubewarden kubewarden-crds
 ```
 
 The command removes all the Kubernetes components associated with the chart, all
```

Comment:
Nit: 

```suggestion
The commands remove all the Kubernetes components associated with the chart, all
```
(there's another one in the following phrase)

Reply:
done, thanks! :)